packages/contracts-bedrock/test/setup/ForkLive.s.sol

```diff
@@ -99,13 +112,22 @@ contract ForkLive is Deployer {
         artifacts.save("PermissionedDelayedWETHProxy", address(permissionedDisputeGame.weth()));
     }
 
+    /// @notice Etches a new Deploy.s.sol contract at a deterministic address, sets up the environment,
+    ///         and deploys new implementations with a "_NextVersion" suffix. This suffix is necessary to avoid
+    ///         naming collisions with the implementations saved above.
+    function _deployNewImplementations() internal {
+        Deploy deploy = Deploy(address(uint160(uint256(keccak256(abi.encode("optimism.deploy"))))));
+        deploy.deployImplementations({ _isInterop: false, _suffix: "_NextVersion" });
+    }
+
```

Comment:
natspec needs an update.

This function became much simpler since the last review,  due to refactoring in Setup.sol